Gemfile

```diff
@@ -44,6 +43,13 @@ group :development, :test do
   gem "quiet_assets", "~> 1.1"
   gem "rspec-rails", "~> 3.2.0"
   gem "rubocop", "~> 0.30"
+  gem "phantomjs", "~> 1.9.7"
+  gem "jasmine-core", "~> 2.0.0"
+  gem "jasmine", "~> 2.0.2"
+  gem "jasmine-rails", "~> 0.10.2"
+  gem "jasmine-jquery-rails", "~> 2.0.2"
+  gem "guard-jasmine", git: "https://github.com/guard/guard-jasmine", branch: "jasmine-2"
```

Comment:
I asked team last week if Jasmine was ok for unit testing javascript. If including guard is causing issues please let me know.